frontend/src/features/properties/map/activity/detail/content/formContent.tsx

```diff
@@ -0,0 +1,53 @@
+import { Input } from 'components/common/form';
+import { useFormikContext } from 'formik';
+import { Col, Row } from 'react-bootstrap';
+import { withNameSpace } from 'utils/formUtils';
+import * as Yup from 'yup';
+
+import { ActivityModel } from '../models';
+import { ActivityTemplateTypes, IActivityFormContent } from './models';
+export const formContent = new Map<ActivityTemplateTypes, IActivityFormContent>([
+  [
+    ActivityTemplateTypes.SURVEY,
+    {
+      header: 'Survey',
+      initialValues: { test: '' },
+      EditForm: () => (
+        <>
+          <b>TEST SURVEY FORM</b>
+          <Row>
+            <Col>
+              <Input field={withNameSpace('activityData', 'test')}></Input>
+            </Col>
+          </Row>
+        </>
+      ),
```

Comment:
This is a new pattern, lets go over this on our dev meeting

Reply:
agree, kinda would like to not inline the EditForm here but load it from a file. Can discuss further in the meeting

Reply:
yes, loading it from a file would be my recommendation. Done here since I'm not actually implemented a real form.